src/sql-parser/src/parser.rs

```diff
@@ -2897,10 +2897,15 @@ impl<'a> Parser<'a> {
         })
     }
 
-    fn parse_create_sink(&mut self) -> Result<Statement<Raw>, ParserError> {
+    fn parse_create_sink(&mut self, with_name: bool) -> Result<Statement<Raw>, ParserError> {
         self.expect_keyword(SINK)?;
         let if_not_exists = self.parse_if_not_exists()?;
```

Comment:
I don't think it really makes sense to parse this if there is no name.

Reply:
What is the behavior of `IF NOT EXISTS` on an `EXPLAIN CREATE SINK` statement? I vaguely remember a conversation around this but don't remember the details.

Reply:
It's ignored. I'm ambivalent about whether we should hard reject it, leave the situation as is (not in docs but allowed), or document explicitly that it's ignored.

Reply:
I would expect the user flow to be something like, they write a create statement and prefix it with say `EXPLAIN VALUE SCHEMA` and run it to see if the schema looks good. So, we should deviate as less as possible from the actual create statement? I would bias towards leaving it as is and not explicitly hard reject.